layout/_partial/archive-post.ejs

```diff
@@ -34,7 +34,7 @@
             <% } %>
             <li class="archive-post archive-day" data-date="<%= post.date.format('YYYYMMDD') %>">
                 <a class="archive-post-title" href="<%- url_for(post.path) %>"><%= post.title || '(' + __('post.no_title') + ')' %></a>
-                <span class="archive-post-date"><%= ' - ' + post.date.locale(page.lang).format(__('date_format')).toLowerCase() %></span>
+                <span class="archive-post-date"><%= ' - ' + post.date.locale(page.lang).format(__('date_format')) %></span>
```

Comment:
revert this and let's focus on the gitment feature :)

Reply:
That's okay, reverted.